File: crates/deslop/tests/fixtures/fused-golden-csharp/verbatim_a.cs
namespace Golden.Verbatim;

public static class WeightedTotals
{
    public static int Accumulate(int[] values, int floor)
    {
        int total = 0;
        foreach (int value in values)
        {
            if (value > floor)
            {
                total = total + value * 2;
            }
            else
            {
                total = total - 1;
            }
        }

        return total;
    }
}

File: crates/deslop/tests/fixtures/fused-golden-php/verbatim_a.php
<?php

function accumulate(array $values, int $floor): int
{
    $total = 0;
    foreach ($values as $value) {
        if ($value > $floor) {
            $total = $total + $value * 2;
        } else {
            $total = $total - 1;
        }
    }

    return $total;
}
